### PR TITLE
Fix golint issues in pkg/apis/batch/validation

### PR DESCRIPTION
**Pull request by hvaara**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
This PR fixes golint issues in `pkg/apis/batch/validation`

**Which issue(s) this PR fixes**:
Ref #68026

**Special notes for your reviewer**:
None

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

/sig testing
/priority backlog
/area code-organization

### CODE REVIEW

File: pkg/apis/batch/validation/validation.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ import (
 
 // TODO: generalize for other controller objects that will follow the same pattern, such as ReplicaSet and DaemonSet, and
 // move to new location.  Replace batch.Job with an interface.
-//
+
 // ValidateGeneratedSelector validates that the generated selector on a controller object match the controller object
 // metadata, and the labels on the pod template are as generated.
```
Comment (draveness) [inline pkg/apis/batch/validation/validation.go]: What about this?

```
// ValidateGeneratedSelector validates that the generated selector on a controller object match the controller object
// metadata, and the labels on the pod template are as generated.	// metadata, and the labels on the pod template are as generated.
//
// TODO: generalize for other controller objects that will follow the same pattern, such as ReplicaSet and DaemonSet, and
// move to new location.  Replace batch.Job with an interface.	// move to new location.  Replace batch.Job with an interface.
```

Comment (hvaara) [reply pkg/apis/batch/validation/validation.go]: Thanks for the review @draveness! 🎉 
That makes more sense. I updated it.

**Review by draveness — APPROVED** [review]
/lgtm

Thanks for doing this

**Comment by hvaara** [comment]
@lavalamp can you PTAL when you have time?

**Pull request merged**